strato-run.sh

```diff
@@ -78,6 +78,7 @@ then
     export BLOC_DOC_URL=${BLOC_DOC_URL:-http://localhost/docs/?url=/bloc/v2.1/swagger.json}
     export STRATO_URL=${STRATO_URL:-http://localhost/strato-api/eth/v1.2}
     export STRATO_DOC_URL=${STRATO_DOC_URL:-http://localhost/docs/?url=/strato-api/eth/v1.2/swagger.json}
+    export CIRRUS_URL=${CIRRUS_URL:-http://localhost/cirrus/search}
```

Comment:
Kind of a shame that we have both `CIRRUS_URL` and `cirrusurl` as env vars. 
Does is make sense that they have different values? 
If that does make sense, is there a better name for them to clarify what they're for and why they're different?